float64[] jointValues
StaubliMovementParams params